<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>566</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>566</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/res/settings.png</normaloff>:/icons/res/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Scanning options</string>
           </property>
           <widget class="QCheckBox" name="checkBox_Recursive">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>81</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Recursive</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Visual Basic detection</string>
           </property>
           <widget class="QCheckBox" name="checkBox_RawVB5">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>151</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Raw scan for &quot;VB5!&quot; magic</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>5</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Leaks</string>
           </property>
           <widget class="QCheckBox" name="checkBox_ShowMethodNameLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>80</y>
              <width>151</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Method Name Pointers Leak</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_ShowResdescrtblLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>RESDESCTBL Leak</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_ShowResourceMajorLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>241</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>RESOURCES Major Leak (2 bytes max.)</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_ShowComObjectLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>COM Object Leak</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_ShowMethodPointersLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>151</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Method Pointers Leak</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_ShowProjectPathLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>120</y>
              <width>151</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Project Path Leak (unsure)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_Summary">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>6</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Summary</string>
           </property>
           <widget class="QCheckBox" name="checkBox_ProjectPath">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Project Path (*.vbp)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_OLBPaths">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>External type/object library paths</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_MethodNames">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Method Names</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_ObjectNames">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>80</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Object Names</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_addCompilerLeftovers">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Compiler Leftovers (IdeData)</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_addRich">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>120</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>RICH Signature</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_Import">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>140</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>VB Import</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_PeImport">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>160</y>
              <width>231</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>PE Import</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_Extra">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>8</horstretch>
             <verstretch>3</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Extra</string>
           </property>
           <widget class="QCheckBox" name="checkBox_OptimizeLeak">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>151</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Optimize Leaks</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="labelCharset">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>50</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Charset:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>46</y>
              <width>171</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>frame_2</zorder>
     <zorder>frame</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSettingsOK">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
